vendor/src/github.com/docker/engine-api/types/types.go

```diff
@@ -274,6 +274,19 @@ type ExecStartCheck struct {
 	Tty bool
 }
 
+type HealthcheckResult struct {
+	Start, End time.Time
+	ExitCode   int    // 0=healthy, 1=unhealthy, 2=starting
+	Output     string // Output from last check
```

Comment:
> Why would it round-trip?

Usually, when we send it over engine API.

> Go will replace the bad data with unicode replacement characters

What constitutes "bad data"? This is the output of a command. What if the output is latin-1 or gb2312? What if it is image data or a binary data?